homeassistant/components/fan/reproduce_state.py

```diff
@@ -73,18 +82,39 @@ async def _async_reproduce_state(
     if state.state == STATE_ON:
         # The fan should be on
         if cur_state.state != STATE_ON:
-            # Turn on the fan at first
-            service_calls[SERVICE_TURN_ON] = service_data
-
-        for attr, service in ATTRIBUTES.items():
-            # Call services to adjust the attributes
-            if attr in state.attributes and not check_attr_equal(
-                state.attributes, cur_state.attributes, attr
-            ):
-                data = service_data.copy()
-                data[attr] = state.attributes[attr]
-                service_calls[service] = data
-
+            # Turn on the fan with all the speed and modes settings.
```

Comment:
Can you help me follow/understand why `SIMPLE_ATTRIBUTES` is not used when going from off to on?